gcp_cost_report/main.py

```diff
@@ -18,14 +19,16 @@
 PROJECT_ID = os.getenv('GCP_PROJECT')
 BIGQUERY_BILLING_TABLE = os.getenv('BIGQUERY_BILLING_TABLE')
 QUERY_TIME_ZONE = os.getenv('QUERY_TIME_ZONE') or 'UTC'
+SLACK_MESSAGE_MAX_CHARS = 2400
```

Comment:
Where did you get this number from? This page says 40,000 chars, https://api.slack.com/changelog/2018-04-truncating-really-long-messages.

If we have to split, can we sort with any compute at the top, and then post any additional messages in the thread?

Reply:
The error message you get when hitting the API is that the message cannot exceed 2000 characters. I'm not exactly sure why though, since I set the limit to 3000 and it would sometimes work. So the rule seems inconsistent. I set it to 2400 and it consistently worked. But it actually is 2000.

I've just changed it in the update to this PR.